### ISSUE TITLE
Encoding variable size input

### ISSUE THREAD

**Issue by mathysgrapotte** [mathysgrapotte/stimulus#12]
Tests for encoding variable length elements fails at the moment, fix is for encode_all to return a list of np arrays instead of a stacked numpy array. 

Later on, in the data loader, sequences should be padded and a mask should be returned according to the padding positions

Then, we can use said mask in the loss computation to prevent the model from learning these values 
```
mask = (seqs != 0) // sequences are 0 padded
output = model(seqs) 
loss_fn = nn.CrossEntropyLoss(reduction='none')
loss = loss_fn(output, target)
masked_loss = loss.masked_select(mask).mean()
```

**Comment by mathysgrapotte** [comment]
Fixed in #9
